src/components/ToolIcon.scss

```diff
@@ -83,3 +83,12 @@
     box-shadow: none;
   }
 }
+
+.ToolIcon__keybinding {
+  position: absolute;
+  bottom: 2px;
+  right: 2px;
```

Comment:
```suggestion
  right: 4px;
```

I think it looks a bit better like this.. or maybe `3px`.. WDYT?

Reply:
At the moment they look to tight to the border.

Reply:
used 3px